@navikt/core/react/src/tabs/parts/tablist/useTabList.ts

```diff
@@ -48,9 +48,16 @@ export function useTabList() {
         End: lastTab,
       };
 
+      const shouldDoAction = !(
```

Comment:
burde den renames til noe som `noModifiers` ? 🤔

Reply:
Forgot to fix in this PR, but made them on main 🚀